lib/bundler/cli.rb

```diff
@@ -166,6 +166,17 @@ def check
       Check.new(options).run
     end
 
+    desc "remove [GEMS] [OPTIONS]", "Removes the gem from the Gemfile"
+    long_desc <<-D
+      Removes the specified gems from the Gemfile in the current directory. If a gem is not found, Bundler prints a error message and if gem could not be removed due to any reason Bundler displays warning.
```

Comment:
> in the current directory

This is not necessarily true, it should remove the gems from whichever Gemfile is configured as well.

Reply:
> and if gem

`the gem`

Reply:
> any reason Bundler displays warning

`any reason, Bundler will display a warning`